### ISSUE TITLE
Refactor CSS/SASS/LESS imports

### ISSUE THREAD

**Issue by leebenson** [reactql/kit#12]
There are currently differences in the way CSS, LESS and SASS `@import` statements resolve, causing unexpected behaviour when using `~` to represent `node_modules`, and relative path names that don't follow the expected Webpack setting.

Needs proper testing.

**Comment by Bartekus** [comment]
Nice work on the refactor @leebenson, I've tested this by successfully integrating Grommet on #18 
Adding `@import '~grommet/scss/vanilla/index';` to the styles.scss works exactly as expected!
Thank you!

**Comment by Extra-lightwill** [comment]
Have you had time to make any progress re: https://github.com/reactql/cli/issues/30 ?  

If so, what would be the recommended way to import sass partials?

**Comment by leebenson** [comment]
@Extra-lightwill - sure have, please see https://github.com/reactql/kit/issues/13

TL;DR - Put 'global' `@import` statements inside a `style.global.css` file and import that instead. It turns out that `:global` should only be used for 'inline' statements. See [this comment](https://github.com/reactql/kit/issues/13#issuecomment-297260139) for a more thorough explanation.

**Comment by leebenson** [comment]
@Extra-lightwill - as for a recommend way to import SASS partials, this should work just like any other SASS import. e.g.:

```scss
// if you're importing locally, from your own source
@import 'src/path/to/partial;

// if you're importing from a `node_modules` npm
@import '~module/path/to/partial'
```
